git-tar/function/ops.go

```diff
@@ -43,21 +43,22 @@ func parseYAML(pushEvent sdk.PushEvent, filePath string) (*stack.Services, error
 }
 
 func fetchTemplates(filePath string) error {
-	templateRepos := []string{"https://github.com/openfaas/templates", "https://github.com/openfaas-incubator/node8-express-template.git", "https://github.com/openfaas-incubator/golang-http-template.git"}
+	if _, err := os.Stat(filePath + "/template"); os.IsNotExist(err) {
+		templateRepos := []string{"https://github.com/openfaas/templates", "https://github.com/openfaas-incubator/node8-express-template.git", "https://github.com/openfaas-incubator/golang-http-template.git"}
```

Comment:
Could you use an error group for this instead so that they are executed in parallel? Look at the way this was done by @AkihiroSuda in the of-builder component.

Reply:
Referenced that but I don't know why it wont make that comment outdated